cts/cts-fencing.in

```diff
@@ -356,6 +382,28 @@ class Test(object):
         if args['stdout_negative_match'] != "" and output.count(args['stdout_negative_match']) != 0:
             raise OutputFoundError(output)
 
+        if args['validate']:
+            if os.path.exists("%s/cts-fencing.in" % TEST_DIR):
+                rng_file = "xml/api/api-result.rng"
+            else:
+                rng_file = "/usr/share/pacemaker/api-result.rng"
```

Comment:
The schema directory is configurable at both build-time and run-time, so we can't hard-code it. Use os.environ['PCMK_schema_directory'] if it exists (whether running from checkout or not), and the checkout location (as you have it above) or @CRM_SCHEMA_DIRECTORY@ otherwise (I like to define constants for such build options at the top so all the funky substitution syntax is in one place).